src/middleware/auth.middleware.js

```
@@ -0,0 +1,24 @@
+const jwt = require('jsonwebtoken');
+
+const ACCESS_SECRET = process.env.JWT_ACCESS_SECRET;
```

`ACCESS_SECRET` may be undefined. Validate this at startup (or throw an explicit error here) so misconfiguration fails fast instead of making every request return a generic 401 and masking a server-side problem.